src/ILCompiler.Compiler/src/CppCodeGen/CppWriter.cs

```diff
@@ -900,7 +889,7 @@ private String GetCodeForNodeStruct(List<NodeDataSection> nodeDataDivs, Dependen
             return nodeStructDecl.ToString();
         }
 
-        private void AppendFormattedByteArray(CppGenerationBuffer sb, byte[] array, int startIndex, int endIndex)
+        private void GetFormattedByteArray(byte[] array, int startIndex, int endIndex, CppGenerationBuffer sb)
```

Comment:
It would be better to keep the original name - it better expresses what the method does.

Also, you can make the method `static` while you are on it.